opus/application/apps/ui/templates/ui/base.html

```diff
@@ -485,6 +485,17 @@ <h3 class='op-header-text'></h3>
             draggable=False
             close_by_keyboard="esc"
         %}
+        {% include "ui/confirm_modal.html" with
+            id_name="op-close-surfacegeo-widgets"
+            confirm_title="Confirm Close All Surfacegeometry Widgets"
```

Comment:
"Surfacegeometry" and "surfacegeometry" are not very friendly terms for the end user. "Widget" is not a term the user understands. We call them "search fields" in the documentation. Also it won't be clear to them why the widgets are being closed.

Title: "Confirm Close All Surface Geometry Search Fields"

Msg: "Closing the Surface Geometry Target Name search field will also close all target-specific search fields. Are you sure you wish to proceed?"